modules/ROOT/pages/organization.adoc

```diff
@@ -97,9 +114,13 @@ To configure the settings of your organization's private branch of Exchange, edi
 
 === Create a Business Group
 
-If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. To create a business group:
+Business groups are also referred to as organizations (or sub organizations). They reside within the master organization and can contain their own business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
```

Comment:
See line 108 above, should we be using that notation under every major section? in other places in this topic we write it out in prose.

Reply:
Does line 117 make more sense as an intro under Business Groups (we don't have to always have content under a heading, but in this case it seems to belong there more than in this task section).

Reply:
Yes!